x-pack/plugins/observability_solution/logs_explorer/common/locators/logs_explorer/logs_explorer_locator.ts

```diff
@@ -21,16 +21,21 @@ export class LogsExplorerLocatorDefinition implements LocatorDefinition<LogsExpl
   constructor(protected readonly deps: LogsExplorerLocatorDependencies) {}
 
   public readonly getLocation = (params: LogsExplorerLocatorParams) => {
-    const { dataset } = params;
+    const { dataset, columns } = params;
     const dataViewSpec: DataViewSpec | undefined = dataset
       ? {
           id: dataset,
           title: dataset,
         }
       : undefined;
 
+    const discoverColumns = columns?.map((column) => {
+      return column.type === 'document-field' ? column.field : column.name;
```

Comment:
**doubt**: I see why the properties are named differently as one represents ECS fields and the other is for smart fields. However, it's confusing using `name` IMO, as it detaches from the concept of field and could mean anything as a human-readable name or others.
Wdyt about `smartField` or keeping `field` itself? There is already the `type` property for differentiating between different kink of fields, renaming also the props might increase the cognitive shift.

Reply:
Me and @weltenwort did discuss on naming this property `name`. You identified it correctly that the original field represents a `ecs mapping` where as we wanted our custom name to be identified differently from `field` and hence suggestion was to use `name`. So i would not want to keep it as `field`. I don't have a strong pref to have it called `name`, can rename it to `smartField`